### DISCUSSION TITLE
Network error when trying to push information to cloud.

### DISCUSSION THREAD

**Discussion by Rubendbdv** [opengisch/QField#4798] [Bugs]
Greetings

When Pushing information to the cloud on our project we recieve the notification 'Network error, try again' (see attached screenshot). 
We use this project everyday to carry out 'Visual Tree Assessments', where we also make reports for our clients.

In the screenshot you can see that there have been 1291 local changes since the last push/synchronize. 
The same error occured when trying to push before this point (first try was at around a hundred changes), but we continued working in the project, that's why there are so many changes now. 

Synchronising the project or pushing/synchronising with a different account also doesn't work, the same error message appears. 

QField log says:
"Failed to upload delta file, reason:<br>Error transferring https://app.qfield.cloud/api/v1/deltas/ab65aa0a-76df-4b99-863b-fcf58f50e04c/ - server replied: Internal Server Error<br>[QF/unknown_error] QFieldCloud Unknown Error"

![WhatsApp Image 2023-11-23 at 08 29 18_8d1a5fde](https://github.com/opengisch/QField/assets/125654399/44414b63-d053-4bf8-ac99-4ba290ed579e)

We found a previous discussion "Unable to push changes on QField Cloud #2995" which describes the same problem. 

Specifics we use:
-Tablet (Trimble R2) with Qfield (version 3.0.7)
-Desktop (different devices, OS Windows 11) with Qgis 3.32
-Connection with WIFI aswell as Mobile Network
-Qfieldcloud username: "thomasdbdv"
-Project: "Boombeheer"

All help is greatly appreciated.

Sincerely
Ruben

**Comment by BoswachterMarc** [comment]
Ruben, perhaps the same problem here?
https://github.com/opengisch/QField/issues/4796
